modules/tensor_mechanics/test/tests/stress_recovery/patch/tests

```diff
@@ -1,8 +1,22 @@
 [Tests]
-  [./patch]
+  [./patch_small_strain]
     type = 'Exodiff'
     input = 'patch.i'
     exodiff = 'patch_out.e'
     mesh_mode = replicated
+    max_parallel = 1
```

Comment:
How about add some comments: The restriction will be removed once #12036 is resolved.  I can come back these examples  when I get #12036 done.

Reply:
Would you like this comment as part of the requirements like or simply as a comment following the max_parallel restriction?